content/en/blog/_posts/2021-08-20-seccomp-default.md

```diff
@@ -0,0 +1,257 @@
+---
+layout: blog
+title: "Hardening Kubernetes security by enabling seccomp by default"
```

Comment:
One concern I have is how to communicate the difference between:
- you can configure your cluster so that seccomp is on by default
- Kubernetes 1.22 changed a cluster-wide default. Seccomp is enabled by default if you upgrade.

The extra challenge is to avoid confusing readers who only skim the list of articles and don't click through to read this one.

Reply:
How about making the title more explicit: 
```suggestion
title: "Enable seccomp for all workloads with a new v1.22 alpha feature"
```